docs/source/deploy.rst

```diff
@@ -288,7 +288,8 @@ The service will ensure Voilà is automatically restarted on startup:
         [Service]
         Type=simple
         PIDFile=/run/voila.pid
-        ExecStart=voila --no-browser voila/notebooks/basics.ipynb
+        ExecStart=/home/ubuntu/.venv/bin/voila --no-browser voila/notebooks/basics.ipynb
```

Comment:
Does this mean that voila was installed by the `ubuntu` user? Or with `pip install -r requirements.txt` as mentioned in https://voila.readthedocs.io/en/latest/deploy.html#steps?